WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -57,6 +56,20 @@ class HelpAndSupportViewController: UIViewController {
     ///
     var displaysDismissAction = false
 
+    /// Custom help center web page's URL
+    /// If non-nil this web page is launched instead of ZenDesk
```

Comment:
super nit:
```suggestion
    /// If non-nil this web page is launched instead of Zendesk
```

Reply:
Nice catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/7553/commits/1492f5b1df2d9f354d108cab5ca9d3fa64d905e8